src/components/common/Breadcrumb.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div>
+    <v-breadcrumbs :items="items">
+      <template v-slot:divider>
+        <v-icon>mdi-home</v-icon>
```

Comment:
Good idea here. Why not make the icon dynamic as well while you're at it?, let the user add the icon as a prop as well, and give it a default value incase they forget